From cb21b3f03cb11c9d6cde60bdb32aa2ad880e956c Mon Sep 17 00:00:00 2001
From: Kuninori Morimoto <kuninori.morimoto.gx@renesas.com>
Date: Wed, 20 Dec 2017 01:38:26 +0000
Subject: [PATCH 246/334] ASoC: soc-dai.h: remove symmetric_xxx from
 snd_soc_dai

ALSA SoC has some duplicate parameter.
snd_soc_dai::symmetric_xxx are one of them.

commit f0fba2ad1b6b ("ASoC: multi-component - ASoC Multi-Component
Support") moved "symmetric_xxx" flags from snd_soc_dai to
snd_soc_dai_driver. No one is using snd_soc_dai::symmetric_xxx now.
Let's remove it

Signed-off-by: Kuninori Morimoto <kuninori.morimoto.gx@renesas.com>
Signed-off-by: Mark Brown <broonie@kernel.org>
---
 include/sound/soc-dai.h | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/include/sound/soc-dai.h b/include/sound/soc-dai.h
index 7f32deb21071..d327aee67600 100644
--- a/include/sound/soc-dai.h
+++ b/include/sound/soc-dai.h
@@ -305,9 +305,6 @@ struct snd_soc_dai {
 	/* DAI runtime info */
 	unsigned int capture_active:1;		/* stream is in use */
 	unsigned int playback_active:1;		/* stream is in use */
-	unsigned int symmetric_rates:1;
-	unsigned int symmetric_channels:1;
-	unsigned int symmetric_samplebits:1;
 	unsigned int probed:1;
 
 	unsigned int active;
-- 
2.24.1

